pipelines/cpCore.yaml

```diff
@@ -139,12 +139,31 @@ tasks:
       python: |
         from lsst.analysis.tools.atools import *
 
-  analyzeFlatDetCore:
+  analyzeFlatCore:
     class: lsst.analysis.tools.tasks.VerifyCalibAnalysisTaskByFilter
     config:
-      connections.outputName: cpFlatDetCore
+      connections.outputName: cpFlatCore
       connections.data: verifyFlatResults
 
+      atools.flatMeanPerAmp: CalibStatisticFocalPlanePlot
+      atools.flatMeanPerAmp.produce.plot.addHistogram: true
+      atools.flatMeanPerAmp.quantityKey: FLAT_MEAN
+      atools.flatMeanPerAmp.unit: electron
+
+      atools.flatNoisePerAmp: CalibStatisticFocalPlanePlot
+      atools.flatNoisePerAmp.produce.plot.addHistogram: true
+      atools.flatNoisePerAmp.quantityKey: FLAT_NOISE
+      atools.flatNoisePerAmp.unit: electron
+
+      python: |
+        from lsst.analysis.tools.atools import *
+
+  analyzeFlatDetMergeCore:
+    class: lsst.analysis.tools.tasks.VerifyCalibAnalysisTaskByFilter
```

Comment:
This is calling the existing task that only has dimensions `["instrument", "physical_filter"]`.  We need a new task that has input data dimensions of `["instrument", "physical_filter", "detector"]`.

Reply:
I think this means we need to add one more task with that set of dimensions.  This all needs to be cleaned up next year to avoid this problem.